File: src/main/java/com/gmail/borlandlp/minigamesdtools/arena/team/lobby/spectator/SpectatorLobby.java
package com.gmail.borlandlp.minigamesdtools.arena.team.lobby.spectator;

import com.gmail.borlandlp.minigamesdtools.arena.ArenaPhaseComponent;
import org.bukkit.entity.Player;

import java.util.Set;

public interface SpectatorLobby extends ArenaPhaseComponent {
    void addPlayer(Player player);
    void removePlayer(Player player);
    Set<Player> getPlayers();
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/arena/team/lobby/spectator/SpectatorLobby.kt
package com.gmail.borlandlp.minigamesdtools.arena.team.lobby.spectator

import com.gmail.borlandlp.minigamesdtools.arena.ArenaPhaseComponent
import org.bukkit.entity.Player

interface SpectatorLobby : ArenaPhaseComponent {
    fun addPlayer(player: Player)
    fun removePlayer(player: Player)
    val players: MutableList<Player>
}